corehq/motech/dhis2/dhis2_config.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from couchdbkit.ext.django.schema import DocumentSchema, ListProperty, StringProperty, SchemaProperty, \
+    SchemaListProperty
+
+from corehq.motech.dhis2.const import DHIS2_EVENT_STATUSES, DHIS2_EVENT_STATUS_COMPLETED
+
+
+def recurse_subclasses(cls):
```

Comment:
To avoid duplication, could you move `recurse_subclasses`, `ValueSource` and its subclasses out of both here and openmrs_config.py, up one directory into the `motech` directory? It might make sense to put it in the `models.py` module, or create a new `value_source.py` module.